lib/common_test/src/test_server_ctrl.erl

```diff
@@ -3908,18 +3920,31 @@ run_test_case1(Ref, Num, Mod, Func, Args, RunInit,
     %% status count on the main process (status of parallel test cases
     %% is updated later by the handle_test_case_io_and_status/0 function)
     case {RunInit,Status} of
-	{skip_init,_} ->			% conf doesn't count
-	    ok;
-	{_,ok} ->
-	    put(test_server_ok, get(test_server_ok)+1);
-	{_,failed} ->
-	    put(test_server_failed, get(test_server_failed)+1);
-	{_,skip} ->
-	    {US,AS} = get(test_server_skipped),
-	    put(test_server_skipped, {US+1,AS});
-	{_,auto_skip} ->
-	    {US,AS} = get(test_server_skipped),
-	    put(test_server_skipped, {US,AS+1})
+        {skip_init,_} ->			% conf doesn't count
+            ok;
+        {_,ok} ->
+            case Loc of
+                [{Module, _}] when Module =/= ct_framework ->
+                    put(test_server_total_time, get(test_server_total_time) + Time);
+                _ -> not_a_test_fun
```

Comment:
in future, avoid mixing styles in single case block.
- if there is a single clause with broken line after `->` then be break line for all clauses in that case block